src/components/molecules/NewsFlashFilterPanel.tsx

```diff
@@ -41,59 +41,65 @@ const useStyles = makeStyles({
   },
 });
 
-export const NewsFlashFilterPanel: React.FC = () => {
+interface IProps {}
+const NewsFlashFilterPanel: FC<IProps> = () => {
   const classes = useStyles();
   const { t } = useTranslation();
   const store: RootStore = useStore();
-  const [activeFilter, setActiveFilter] = useState(SourceFilterEnum.all);
 
   return (
     // will be convert to filterButtonsMap instead of code duplicate for the filter button
     <Box className={classes.container}>
       <AnyWayButton
         className={
-          activeFilter === SourceFilterEnum.all ? classnames(classes.buttonClicked, classes.button) : classes.button
+          store.newsFlashActiveFilter === SourceFilterEnum.all
+            ? classnames(classes.buttonClicked, classes.button)
+            : classes.button
         }
         onClick={() => {
-          store.filterNewsFlashCollection(SourceFilterEnum.all);
-          setActiveFilter(SourceFilterEnum.all);
+          store.setActiveNewsFlashFilter(SourceFilterEnum.all);
         }}
       >
         <Typography.Title2>{t('filterPanel.all')}</Typography.Title2>
       </AnyWayButton>
       <AnyWayButton
         className={
-          activeFilter === SourceFilterEnum.ynet ? classnames(classes.buttonClicked, classes.button) : classes.button
+          store.newsFlashActiveFilter === SourceFilterEnum.ynet
```

Comment:
Proper usage of classNames (keep this casing, like in the classNames docs) would be:
```
classNames(classes.button, { 
  classes.buttonClicked : store.newsFlashActiveFilter === SourceFilterEnum.ynet
})
```

Reply:
Also `active` would be a better name than "buttonClicked"